### DISCUSSION TITLE
[Q&A] lobe-chat 的回复能否逐行一点点的显示回复？

### DISCUSSION THREAD

**Discussion by se7enmuting** [lobehub/lobehub#831] [General Question] [answered]
就像 chatGPT 官方或者 NEXT 那样，是逐行一点点的显示回复。
我是自己用docker部署的，发现回复很慢，要全部回复内容生成完毕再一起显示回复，有没有什么设置或者部署的时候的环境参数可以让 lobe-chat 一点点显示回复吗？

**Comment by arvinxx** [comment, marked as answer]
如果是流式的问题，详见：https://github.com/lobehub/lobe-chat/discussions/531

**Comment by se7enmuting** [comment]
感谢，解决了，原来是nginx反代的锅。。。
